test/unit/builtins/string.spec.ts

```diff
@@ -56,11 +56,11 @@ test.each([
     { inp: "hello test", searchValue: "test", replaceValue: "" },
     { inp: "hello test", searchValue: "test", replaceValue: "world" },
     { inp: "hello test", searchValue: "test", replaceValue: "%world" },
+    { inp: "hello test", searchValue: " test", replaceValue: "." },
```

Comment:
Could you also add some tests containing some of these other non-alphanumerics (both as search and replace values)?